ledger-core/server/server.go

```diff
@@ -24,6 +24,10 @@ func NewVirtualServer(cfgPath string) Server {
 	return insapp.New(cfgPath, virtual.AppFactory)
 }
 
+func NewVirtualMultiServer(cfgPath string, multiFn insapp.MultiNodeConfigFunc) Server {
```

Comment:
this should no be Virtual. It reads configs and starts N nodes. There could be LMN there.